docusaurus/docs/installation.md

```diff
@@ -146,7 +172,7 @@ yarn install --force
 yarn reassure
 ```
 
-## CI integration
+### Integration
 
 As a final setup step you need to configure your CI to run the performance testing script and output the result.
 For presenting output at the moment we integrate with Danger JS, which supports all major CI tools.
```

Comment:
Can we redact that, so that it mentions `dangerfile.reassure.ts`?

Reply:
I can reiterate so that it's there as well, but i's been mentioned already above in the same encompassing section.

Reply:
<img width="219" alt="Screenshot 2023-02-06 at 15 41 35" src="https://user-images.githubusercontent.com/18323849/217001857-050be6b7-7de1-441e-afd4-c53fe23dcff9.png">

Like in the docs